src/ansys/aedt/core/maxwell.py

```diff
@@ -2173,6 +2125,16 @@ def create_setup(self, name="MySetupAuto", setup_type=None, **kwargs):
         setup.update()
         return setup
 
+    def _create_boundary_object(self, name, props, boundary_type):
+        if boundary_type in ["Force", "Torque", "Matrix", "LayoutForce"]:
+            bound = MaxwellParameters(self, name, props, boundary_type)
+        else:
+            bound = BoundaryObject(self, name, props, boundary_type)
+        if bound.create():
+            self._boundaries[bound.name] = bound
+            return bound
+        return False
```

Comment:
```suggestion
            self.logger.info(f"Boundary {name} has been correctly created.")
            self._boundaries[bound.name] = bound
            return bound
        self.logger.error(f"Boundary {name} was not created.")
        return False
```
In the previous method to create boundary, logging was performed. We should also have it here.